changelog_wrap.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: base
+permalink: /changelog/
+---
+
+{% include_relative CHANGELOG.md %}
```

Comment:
Why the `include_relative` here instead of adding the front matter to the changelog.md?

Reply:
I thought it might be better if the actual changelog remained free of the front matter. As it is, I just copied it over from the dashboard repo. But I'm also ok adding the cruft so we can avoid the hack.

Reply:
I'm fine with this, it works.  I was more curious if there was an issue this was avoiding. Maybe organizing the changelog the same way blog posts are would be helpful?

Reply:
🤷‍♂ 

@alorenzo175 do you care how this change log is organized and where it lives?

Reply:
nope